tests/py_localhost/server.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
+# Minor change based on the example from hyper h2 server
+# https://github.com/python-hyper/h2/blob/master/examples/asyncio/asyncio-server.py
+
+"""
+
+asyncio-server.py
+
+~~~~~~~~~~~~~~~~~
+
+
+A fully-functional HTTP/2 server using asyncio. Requires Python 3.5+.
+
+
+This example demonstrates handling requests with bodies, as well as handling
+
+those without. In particular, it demonstrates the fact that DataReceived may
+
+be called multiple times, and that applications must handle that possibility.
+
+"""
+
+import asyncio
+
+import io
+
+import json
+
+import ssl
+
+import collections
+
+from typing import List, Tuple
+from urllib import response
+
+
+from h2.config import H2Configuration
+
+from h2.connection import H2Connection
+
+from h2.events import (
+
+    ConnectionTerminated, DataReceived, RemoteSettingsChanged,
+
+    RequestReceived, StreamEnded, StreamReset, WindowUpdated
+
+)
```

Comment:
trivial: please fix the spacing in this file, it's hard to look at. Every other line is blank. I think something went wrong when you copy/pasted it off github

try this:
repeatedly slam find-replace: `\n\n` -> `\n`
then autoformat